cli/src/main/java/com/expediagroup/streamplatform/streamregistry/cli/command/Apply.java

```diff
@@ -71,12 +78,23 @@
     protected String type;
     @Option(names = "--configuration", required = true, converter = ObjectNodeConverter.class)
     protected ObjectNode configuration;
+    @Option(names = "--security", converter = ObjectNodeConverter.class)
+    protected ObjectNode security = mapper.createObjectNode();
 
     @Override
     public List<Event<?, ?>> events() {
       return Collections.singletonList(specification(getEntityOptions().key(), getSpecification()));
     }
 
+    public Map<String, List<Principal>> convertSecurityMap(ObjectNode security) {
```

Comment:
I thought we said there'd be a wrapper around the role, so `Map<Role, List<Principal>>`. @teabot ?

Reply:
@nahguam @teabot - the CLI converts the incoming entities directly into the state model. The state model uses `string -> List<Principal>` because of the restriction in Avro for string keys in Maps.